content/manager/security.md

```diff
@@ -4,436 +4,701 @@ title: Security
-Available in the premium edition only
-{{% /gsNote %}}
-
-## Cloudify CLI: cfy commands
-cfy commands that connect to the REST service (e.g. "deployments create") send username and password in a header on each
-request. Setting these values is done once, via these environment variables:
-
- - CLOUDIFY_USERNAME
- - CLOUDIFY_PASSWORD
+Cfy commands automatically add the "Authorization" header to each request sent to the manager, containing the username
+and password of the current user. Setting the user credentials is done once, by exporting these environment variables:
 
-### SSL CLI configuration
+ - export CLOUDIFY_USERNAME=my_username
+ - export CLOUDIFY_PASSWORD=my_password
 
-#### Connect with a manager secured by SSL:
-  - After bootstrap, every request to the secured manager will be on port 443, using the 'https' protocol.
-  - The `use` command requires specifing the port option to be 443
-      <br>e.g.
-      {{< gsHighlight  bash  >}}
-        'use -t <manager-ip-address> --port 443'
-      {{< /gsHighlight >}}
+<br>
+#### Client SSL configuration
+**Connecting to the manager over SSL**<br>
+After bootstrap, requests sent to the manager must use port 443 and the 'https' protocol. <br>
+The `cfy use` command requires specifying the port option to be 443:
+{{< gsHighlight  bash  >}}
+cfy use -t <manager-ip-address> --port 443
+{{< /gsHighlight >}}
 
-#### Manager's certificate verification:
-By default, the manager's certificate will be verified.<br>
-There are two environment variables that can be set:
+**Verifying the manager's certificate**<br>
+By default, the CLI attempts to validate the manager's certificate using public CAs (Certificate Authorities).
+The following environment variable can alter that behavior:
 
-  - `CLOUDIFY_SSL_TRUST_ALL` - set to any non empty value to avoid certificate verification.
-  - `CLOUDIFY_SSL_CERT` - set the path to a copy of the manager's certificate in case verification is wanted (trust all is not set) and the manager's certificate is a self-signed certificate.
+ * CLOUDIFY_SSL_CERT - if public CAs should not be used (e.g. when working offline or
+ validating a self-singed certificate), a client-copy of the server's certificate can be used instead.
+To enable this feature set CLOUDIFY_SSL_CERT to the client's local path to the manager's certificate.
+ * CLOUDIFY_SSL_TRUST_ALL - to accept the manager's certificate without validation, set this environment variable to
+ True (or any non-empty value)
 
+<br>
+### Web UI
+Credentials for all requests sent from the Web UI are set at Login. Do note that this client only supports
+authentication by username and password. Other authentication methods (e.g. tokens) are currently not supported.
 
-{{% gsNote title="Note" %}}
-This client only supports authentication with username-password
-{{% /gsNote %}}
-
-## Other REST clients
-other REST clients (e.g. cURL) must explicitly add credentials to each request.<br>
+<br>
+### Other REST Clients
```

Comment:
Maybe we want to show a short snippet of how our python REST client can be configured?